client/components/AuthForm.js

```diff
@@ -24,14 +24,27 @@ const AuthForm = props => {
           </label>
           <input name="password" type="password" />
         </div>
+        <div>
```

Comment:
this makes it so that you request a user enter in their first and last name to log in (but we only want that w hen they sign up) so you can add a ternary here to check if the `formName`
if the `formName` is `logIn` then you just want to show the email and password fields
but if `formNam` is `signup` then you want to show all of them

Reply:
Hey I fixed this, but I'm not sure if this is how it should be